src/AgDatabaseMove.cs

```diff
@@ -73,8 +73,10 @@ public decimal Move(decimal? lastLsn = null)
 
       if(!backupList.Any())
         throw new BackupChainException("No backups found to restore");
+      
+      var stripedBackupSetChain = StripedBackupSet.GetStripedBackupSetChain(backupList.ToList());

```

Comment:
`backupList` is already an `IList<BackupMetadata>` type.  Why call `.ToList()` on it?

Reply:
I'll take a look, might be a remnant from when had the parameter as `List` type instead of `IEnumerable`